src/Root.tsx

```diff
@@ -0,0 +1,16 @@
+import { Route, HashRouter as Router, Routes } from 'react-router-dom';
+import { App } from './App';
+
+export const Root = () => (
+  <Router>
```

Comment:
I mean that style of routing (hm react_tabs-with-router):
` <Route path="/" element={<App />}>
            <Route index element={<HomePage />} />
            <Route path="/home" element={<Navigate to="/" />} />

            <Route path="tabs">
              <Route path=":tabId?" element={<TabsPage />} />
            </Route>

            <Route path="*" element={<ErrorPage />} />
          </Route>
        </Routes>`